Framework/Muon/src/EstimateMuonAsymmetryFromCounts.cpp

```diff
@@ -207,11 +208,21 @@ void EstimateMuonAsymmetryFromCounts::exec() {
   }
   // update table
   Mantid::API::ITableWorkspace_sptr table = getProperty("NormalizationTable");
-  updateNormalizationTable(table, wsNames, norm, methods);
-  setProperty("NormalizationTable", table);
+  if (table) {
+    updateNormalizationTable(table, wsNames, norm, methods);
+    setProperty("NormalizationTable", table);
+  }
   // Update Y axis units
   outputWS->setYUnit("Asymmetry");
 
+  std::string normString =
+      std::accumulate(norm.begin() + 1, norm.end(), std::to_string(norm[0]),
+                      [](const std::string &a, double b) {
+                        return a + ',' + std::to_string(b);
```

Comment:
what is `a` and `b`

Reply:
So I think this is just a fancy way to turn a vector of doubles into a comma separated String of the values. It basically just iterates through the vector turning an entry into a string and adding it to the already existing string. In this case `a` is the already existing string and `b` is the new double value you are trying to add. 

Pseudocode which I think does the same thing would be (C++ syntax may be slightly wrong)

```
std::string normString = std::to_string(norm[0])
for(int i =1 ; i < norm.size(); i++){
    normString += "," + std::to_string(norm[i])
}
```

Reply:
ah ok, that makes sense. I think we just swap the names for the ones you have and I would be happy with it